index.js

```diff
@@ -37,9 +39,9 @@ var MysqlStore = function (options) {
     }
 
     this.cleanup = function() {
-        let now = (new Date()).valueOf() ;
```

Comment:
This bug already fixed with .valueOf()  (it's already returning the value instead the function)
valueOf and getTime are pretty much the same.

You don't see the fix, because @tb01923 hasn't release a new version yet.